### ISSUE TITLE
New tenant creation error

### ISSUE THREAD

**Issue by mizzgoldberg** [timberline-secondary/hackerspace#427]
![Screenshot from 2020-04-12 14-49-25](https://user-images.githubusercontent.com/55294487/79080798-de1f7e80-7ccc-11ea-904a-8e74da62191a.png)

**To Reproduce**
Add tenant - enter name - save

**Comment by tylerecouture** [comment]
````
Environment:


Request Method: POST
Request URL: https://bytedeck.com/admin/tenant/tenant/add/

Django Version: 2.2.12
Python Version: 3.5.9
Installed Applications:
('tenant_schemas',
 'tenant.apps.TenantConfig',
 'grappelli',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.sites',
 'django.contrib.staticfiles',
 'django.contrib.flatpages',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'crispy_forms',
 'django_summernote',
 'bootstrap_datepicker_plus',
 'embed_video',
 'django_select2',
 'jchart',
 'url_or_relative_url_field',
 'import_export',
 'django_celery_beat',
 'colorful',
 'attachments',
 'hackerspace_online',
 'quest_manager',
 'profile_manager',
 'announcements',
 'comments',
 'notifications',
 'courses',
 'prerequisites',
 'badges',
 'djcytoscape',
 'portfolios',
 'utilities',
 'siteconfig')
Installed Middleware:
['tenant_schemas.middleware.TenantMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.security.SecurityMiddleware']



Traceback:

File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py" in _execute
  82.                 return self.cursor.execute(sql)

The above exception (relation "django_migrations" already exists
) was the direct cause of the following exception:

File "/usr/local/lib/python3.5/site-packages/django/db/migrations/recorder.py" in ensure_schema
  67.                 editor.create_model(self.Migration)

File "/usr/local/lib/python3.5/site-packages/django/db/backends/base/schema.py" in create_model
  307.         self.execute(sql, params or None)

File "/usr/local/lib/python3.5/site-packages/django/db/backends/base/schema.py" in execute
  137.                 cursor.execute(sql, params)

File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py" in execute
  99.             return super().execute(sql, params)

File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py" in execute
  67.         return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)

File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py" in _execute_with_wrappers
  76.         return executor(sql, params, many, context)

File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py" in _execute
  84.                 return self.cursor.execute(sql, params)

File "/usr/local/lib/python3.5/site-packages/django/db/utils.py" in __exit__
  89.                 raise dj_exc_value.with_traceback(traceback) from exc_value

File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py" in _execute
  82.                 return self.cursor.execute(sql)

During handling of the above exception (relation "django_migrations" already exists
), another exception occurred:

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py" in wrapper
  606.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  44.         response = view_func(request, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/sites.py" in inner
  223.             return view(request, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py" in add_view
  1645.         return self.changeform_view(request, None, form_url, extra_context)

File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapper
  45.         return bound_method(*args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py" in changeform_view
  1529.             return self._changeform_view(request, object_id, form_url, extra_context)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py" in _changeform_view
  1572.                 self.save_model(request, new_object, form, not add)

File "/app/src/tenant/admin.py" in save_model
  42.         obj.save()

File "/usr/local/lib/python3.5/site-packages/tenant_schemas/models.py" in save
  71.                 self.create_schema(check_if_exists=True, verbosity=verbosity)

File "/usr/local/lib/python3.5/site-packages/tenant_schemas/models.py" in create_schema
  118.                          verbosity=verbosity)

File "/usr/local/lib/python3.5/site-packages/django/core/management/__init__.py" in call_command
  148.     return command.execute(*args, **defaults)

File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py" in execute
  364.         output = self.handle(*args, **options)

File "/usr/local/lib/python3.5/site-packages/tenant_schemas/management/commands/migrate_schemas.py" in handle
  47.             executor.run_migrations(tenants=tenants)

File "/usr/local/lib/python3.5/site-packages/tenant_schemas/migration_executors/base.py" in run_migrations
  61.         self.run_tenant_migrations(tenants)

File "/usr/local/lib/python3.5/site-packages/tenant_schemas/migration_executors/standard.py" in run_tenant_migrations
  9.             run_migrations(self.args, self.options, self.codename, schema_name)

File "/usr/local/lib/python3.5/site-packages/tenant_schemas/migration_executors/base.py" in run_migrations
  31.     MigrateCommand(stdout=stdout, stderr=stderr).execute(*args, **options)

File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py" in execute
  364.         output = self.handle(*args, **options)

File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py" in wrapped
  83.             res = handle_func(*args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/core/management/commands/migrate.py" in handle
  234.             fake_initial=fake_initial,

File "/usr/local/lib/python3.5/site-packages/django/db/migrations/executor.py" in migrate
  117.             state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)

File "/usr/local/lib/python3.5/site-packages/django/db/migrations/executor.py" in _migrate_all_forwards
  147.                 state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)

File "/usr/local/lib/python3.5/site-packages/django/db/migrations/executor.py" in apply_migration
  246.                     self.record_migration(migration)

File "/usr/local/lib/python3.5/site-packages/django/db/migrations/executor.py" in record_migration
  261.             self.recorder.record_applied(migration.app_label, migration.name)

File "/usr/local/lib/python3.5/site-packages/django/db/migrations/recorder.py" in record_applied
  82.         self.ensure_schema()

File "/usr/local/lib/python3.5/site-packages/django/db/migrations/recorder.py" in ensure_schema
  69.             raise MigrationSchemaMissing("Unable to create the django_migrations table (%s)" % exc)

Exception Type: MigrationSchemaMissing at /admin/tenant/tenant/add/
Exception Value: Unable to create the django_migrations table (relation "django_migrations" already exists
)
````

**Comment by mizzgoldberg** [comment]
Tried to create new tenant: Library

Environment:


Request Method: POST
Request URL: https://bytedeck.com/admin/tenant/tenant/add/

Django Version: 2.2.12
Python Version: 3.5.9
Installed Applications:
('tenant_schemas',
 'tenant.apps.TenantConfig',
 'grappelli',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.sites',
 'django.contrib.staticfiles',
 'django.contrib.flatpages',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'crispy_forms',
 'django_summernote',
 'bootstrap_datepicker_plus',
 'embed_video',
 'django_select2',
 'jchart',
 'url_or_relative_url_field',
 'import_export',
 'django_celery_beat',
 'colorful',
 'attachments',
 'hackerspace_online',
 'quest_manager',
 'profile_manager',
 'announcements',
 'comments',
 'notifications',
 'courses',
 'prerequisites',
 'badges',
 'djcytoscape',
 'portfolios',
 'utilities',
 'siteconfig')
Installed Middleware:
['tenant_schemas.middleware.TenantMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.security.SecurityMiddleware']



Traceback:

File "/usr/local/lib/python3.5/site-packages/django/db/backends/base/base.py" in _commit
  240.                 return self.connection.commit()

The above exception (insert or update on table "django_admin_log" violates foreign key constraint "django_admin_log_user_id_c564eba6_fk_auth_user_id"
DETAIL:  Key (user_id)=(3) is not present in table "auth_user".
) was the direct cause of the following exception:

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py" in wrapper
  606.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  44.         response = view_func(request, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/sites.py" in inner
  223.             return view(request, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py" in add_view
  1645.         return self.changeform_view(request, None, form_url, extra_context)

File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapper
  45.         return bound_method(*args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py" in changeform_view
  1529.             return self._changeform_view(request, object_id, form_url, extra_context)

File "/usr/local/lib/python3.5/site-packages/django/db/transaction.py" in __exit__
  240.                         connection.commit()

File "/usr/local/lib/python3.5/site-packages/django/db/backends/base/base.py" in commit
  262.         self._commit()

File "/usr/local/lib/python3.5/site-packages/django/db/backends/base/base.py" in _commit
  240.                 return self.connection.commit()

File "/usr/local/lib/python3.5/site-packages/django/db/utils.py" in __exit__
  89.                 raise dj_exc_value.with_traceback(traceback) from exc_value

File "/usr/local/lib/python3.5/site-packages/django/db/backends/base/base.py" in _commit
  240.                 return self.connection.commit()

Exception Type: IntegrityError at /admin/tenant/tenant/add/
Exception Value: insert or update on table "django_admin_log" violates foreign key constraint "django_admin_log_user_id_c564eba6_fk_auth_user_id"
DETAIL:  Key (user_id)=(3) is not present in table "auth_user".

**Comment by tylerecouture** [comment]
Fixed with PR #429 due to this error:
https://github.com/bernardopires/django-tenant-schemas/issues/490
